usr/share/rear/prep/YUM/default/200_selinux_in_use.sh

```diff
@@ -0,0 +1 @@
+../../RSYNC/GNU/Linux/200_selinux_in_use.sh
```

Comment:
I suspect this has never worked properly with BACKUP_PROG=rsync. Here we have:
https://github.com/rear/rear/blob/e68326f7ef602d2746b0308bebf6db7e4db8f495/usr/share/rear/prep/RSYNC/GNU/Linux/200_selinux_in_use.sh#L27

but nobody creates $TMP_DIR/rsync_protocol in the YUM case. (It gets created for `BACKUP=RSYNC` and `BACKUP=NETFS` by separate scripts, although in one case it seems to be the server protocol version and in the other the client protocol version.)